src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/PetsPerProjectServiceAccountProvider.scala

```diff
@@ -20,4 +20,11 @@ class PetsPerProjectServiceAccountProvider(config: Config) extends SamServiceAcc
     Future(None)
   }
 
+  override def listUsersStagingBucketReaders(userEmail: WorkbenchEmail)(implicit executionContext: ExecutionContext): Future[List[WorkbenchEmail]] = {
+    Future.successful(List.empty[WorkbenchEmail])
```

Comment:
So, for clarification, the staging bucket is not accessible to any users, only the proxy group?

Reply:
Yes, the proxy group should include the user as well

Reply:
face-to-face conversation, makes sense now